streams/src/test/java/org/apache/kafka/streams/integration/KTableSourceTopicRestartIntegrationTest.java

```diff
@@ -100,7 +98,7 @@ public static void closeCluster() {
 
     @BeforeEach
     public void before(final TestInfo testInfo) throws Exception {
-        sourceTopic = SOURCE_TOPIC + "-" + testInfo.getTestMethod().map(Method::getName);
+        sourceTopic = SOURCE_TOPIC + "-" + IntegrationTestUtils.safeUniqueTestName(getClass(), testInfo);
```

Comment:
Note that the test fails without this change.

Reply:
Yeah, thank you! That is known. See https://github.com/apache/kafka/pull/12441#issuecomment-1195516666